operator/v1/types_console.go

```diff
@@ -106,6 +106,36 @@ type ConsoleCustomization struct {
 	// SVG format preferred
 	// +optional
 	CustomLogoFile configv1.ConfigMapFileReference `json:"customLogoFile,omitempty"`
+	// console allow the cluster admin to configure the shown categories.
+	DeveloperCatalog DeveloperConsoleCatalogCustomization `json:"developerCatalog"`
+}
+
+// ConsoleCatalog allow the cluster admin to configure developer catalog.
+type DeveloperConsoleCatalogCustomization struct {
```

Comment:
Are we plan to have additional fields in the `spec.customization. developerCatalog`, besides `categories` ?

Reply:
This might happen, yeah